### PR TITLE
Fix Behat and static analysis

### PR DESCRIPTION
**Pull request by jjanvier**
| Q                    | A
| -------------------- | ---
| Bug fix?             | Y
| New feature?         | N 
| BC breaks?           | N
| CI currently passes? | N
| Tests pass?          | Y
| Scenarios pass?      | running
| Checkstyle issues?*  | N
| PMD issues?**        | N
| Changelog updated?   |
| Fixed tickets        |
| DB schema updated?   |
| Migration script?    |
| Doc PR               |

### CODE REVIEW

File: features/reference-data/variant-group/add_products.feature [hunk 1]
```diff
@@ -47,6 +47,7 @@ Feature: Add products with reference data to a variant group
     And I am on the "SANDAL" variant group page
     And I check the row "heel-yellow-37"
     And I press the "Save" button
+    And I press the "Save" button
```
Comment (grena) [inline features/reference-data/variant-group/add_products.feature]: Why twice ?

File: src/Pim/Component/ReferenceData/LabelRenderer.php [hunk 2]
```diff
@@ -30,6 +30,20 @@ public function render(ReferenceDataInterface $referenceData)
...
+    }
+
+    /**
+     * @param ReferenceDataInterface $referenceData
+     *
+     * @return string
+     */
+    public function getLabelProperty(ReferenceDataInterface $referenceData)
+    {
+        return $labelProperty = $referenceData::getLabelProperty();
```
Comment (skeleton) [inline src/Pim/Component/ReferenceData/LabelRenderer.php]: ?

File: src/Pim/Bundle/ReferenceDataBundle/MongoDB/Normalizer/ReferenceDataNormalizer.php [hunk 3]
```diff
@@ -24,6 +38,10 @@ public function normalize($object, $format = null, array $context = array())
             'code' => $object->getCode()
         ];
 
+        if (null !== $label = $this->renderer->render($object, false)) {
```
Comment (grena) [inline src/Pim/Bundle/ReferenceDataBundle/MongoDB/Normalizer/ReferenceDataNormalizer.php]: We should avoid assignment in control no?

Comment (jjanvier) [inline src/Pim/Bundle/ReferenceDataBundle/MongoDB/Normalizer/ReferenceDataNormalizer.php]: we should avoid `if ($label = $this->renderer->render($object, false)) {`

**Comment by sumbobyboys** [comment]
GTM

**Comment by grena** [comment]
GTM

**Pull request merged**